xml/art_solution_automation.xml

```diff
@@ -0,0 +1,435 @@
+      &susemgr; and &slsreg; for any sort of automation.
+    </para>
+    <para>
+      The major motivation was to improve and simplify the installation of &hana;
+      on &sls;.
+    </para>
+    <remark>toms 2020-06-10: Mention Stack Formulas?</remark>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-terms">
+    <title>Terminology</title>
+    <remark>toms 2020-06-23: add more terms and further explanations?</remark>
+    <variablelist>
+      <varlistentry>
+        <term>Python</term>
+        <listitem>
+          <para>
+            A scripting language. It interacts with the low layers utilities
+            like &crmshell; and several &sap; commands, including &hana; management
+            tools.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&salt; (also &saltstack;)</term>
+        <listitem>
+          <remark>toms 2020-06-10: https://docs.saltstack.com/en/getstarted/</remark>
+          <para>
+            A configuration infrastructure management system written in Python.
+            Due to its modular approach, it is often referred as &saltstack;.
+            <!--&salt; configures and provision the needed machines/nodes.-->
+            &salt; works as a client/server architecture. The server (also called
+            &saltmaster;) acts as a central control unit for the clients
+            (<quote>minions</quote>). The other supported setup option is called masterless.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&salt; Grains</term>
+        <listitem>
+          <para>
+            Static data about &salt; minions. Grains contain information about
+            operating system that is running, the used CPU architecture, and much more.
+            Grains can also be set to assign values to minions.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&salt; Formulas</term>
+        <listitem>
+          <para>
+            Formulas are pre-written &salt; States.
+          </para>
+          <!--
+            Installed:
+            * through a Git repository
+            * through a directory
+          -->
+          <para>
+            For more information about &salt;, refer to the upstream documentation at
+            <link xlink:href="https://docs.saltstack.com"/>.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&saltmaster;</term>
+        <listitem>
+          <para>
+            Manages the infrastructure and the minions within.
+            It can execute commands remotely onto minions and manage their state.
+            The &saltmaster; captures grains sent from minions and decide what to do
+            with this information.
+          </para>
+        </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>&salt; Minions</term>
```

Comment:
Since the explanation is in the singular, should this read "Salt Minion" (singular), too?

Reply:
Good catch. :+1: I'll use the singular, thanks.